user_docs/en/changes.t2t

```diff
@@ -13,10 +13,10 @@ There are also various bug fixes and improvements, notably for braille support a
 
 
 == Important notes ==
-Screen Curtain is no longer compatible when using versions of NVDA older than 2021.2 with Windows newer than Windows 10 21H2 (10.0.19044).
+Screen Curtain is no longer compatible when using versions of NVDA older than 2021.2 with Windows 10 21H2 (10.0.19044) or later.
 This includes Windows 10 Insiders and Windows 11.
 This is due to a change in the Windows Magnification API.
-For security purposes, confirm that the Screen Curtain works properly when using a new version of Windows for the first time.
+For security purposes, get visual confirmation that the Screen Curtain works properly when using a new version of Windows for the first time.

```

Comment:
If #12701 is merged, this isn't necessary, right?

Reply:
I don't think #12701 should be documented as a solution. The screen curtain check could be a false positive, for example, a buffer grab fail like you described in https://github.com/nvaccess/nvda/pull/12701#issuecomment-890696961. Or, Windows might display something different to whatever method the screen capture API uses.

Reply:
We should work through these possible edge cases to gain confidence in the approach of #12701 so that it can be relied on. Otherwise it isn't useful. If we aren't confident in it for 2021.2, then I'd push it back to 2021.3 and ask people to check visually until we have an automated approach.

Reply:
See https://github.com/nvaccess/nvda/pull/12700#discussion_r681345930

Without understanding how the screen capture mechanism works, I don't know if it is possible to reliably future proof the screen curtain check.

Reply:
I would prefer to push it back until that can be researched properly.